### DISCUSSION TITLE
Missed word

### DISCUSSION THREAD

**Discussion by djohnsonbhrp** [deepgram/community#750] [General help] [answered]
Are there any settings I can change to increase the likelihood that short words aren't missed? Unfortunately, our audio source isn't multichannel, and in this case speaker 1 was talking and speaker 2 said "yup" a little softly, but very near to where speaker 1's original sentence ended, and very near to where speaker 1's next sentence began. Any way to account for this?

For Deepgram, the request ID is 79fbd918-64be-4da4-a1d6-1c3e1fea860e and the missed word is at 21:05.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by djohnsonbhrp** [comment]
endpoint was: /v1/listen?model=nova-2&version=latest&punctuate=true&diarize=true&multichannel=true&smart_format=true&utterances=true&paragraphs=true&filler_words=false

**Comment by djohnsonbhrp** [comment]
For Deepgram, there was also some cross-talk around 24:44 as well and it missed a speaker's response.

**Reply by jpvajda** [reply, marked as answer]
@djohnsonbhrp  [keywords](https://developers.deepgram.com/docs/keywords) can help catch words you want Deepgram to process.
